source/reference/limitations.txt

```diff
@@ -75,7 +75,11 @@ discuss your requirements.
 Unsupported Collection Types
 ----------------------------
 
-- Time-series collections are not supported.
+- If you use ``mongosync`` 1.6 or earlier, time series collections are
```

Comment:
[nit]

Consider updating this to use active voice throughout.

Spitballing: 

"In \``mongosync\`` 1.6 and earlier, time series collections do not synchronize between clusters: MongoDB logs collection details and skips those collections."

Reply:
Will do.